lib/app/bloc_observer.dart

```diff
@@ -1,9 +1,15 @@
 import 'package:bloc/bloc.dart';
 
 class SimpleBlocObserver extends BlocObserver {
+  @override
+  void onTransition(Bloc bloc, Transition transition) {
+    super.onTransition(bloc, transition);
+    print(transition);
+    // check event and log, if necessary, analytics event here
+  }
+
   @override
   void onChange(BlocBase bloc, Change change) {
     super.onChange(bloc, change);
-    print('${bloc.runtimeType} $change');
```

Comment:
If no side effect/log/print is added, then we could completely remove `onChange` and only rely on `onTransition`

Reply:
i removed it but i didn't commit that! 🤦‍♂️ 
Thanks  💙

Reply:
@jorgecoca following this topic. I know there is a linter rule in the VeryGoodAnalysis to avoid using print statements in production code. So I imagine there are never print statements in the production code at VGV. However, is `debugPrint` used?